src/pushsource/_impl/schema/staged-schema.yaml

```diff
@@ -104,6 +104,53 @@ definitions:
                 type:
                 - boolean
                 - "null"
+            release_notes:
+                $ref: "#/definitions/optional_string"
+
+            usage_instructions:
+                $ref: "#/definitions/optional_string"
+
+            recommended_instance_type:
+                $ref: "#/definitions/optional_string"
+
+            marketplace_entity_type:
+                $ref: "#/definitions/optional_string"
+
+            scanning_port:
+                type:
+                    - number
+                    - "null"
+                minimum: 0
```

Comment:
Connecting to a comment above, what does 0 mean actually?

Reply:
Actually, it does not really matter what this means, only if it matches what the AWS API accepts.  Accepting 0 is correct if AWS also accepts 0.

Reply:
As it accepts 0 so I am leaving this unchanged.